src/datajudge/db_access.py

```diff
@@ -632,43 +632,63 @@ def get_row_count(engine, ref, row_limit: int = None):
     return result, [selection]
 
 
-def _column_aggregate(engine, ref, column_operator):
+def _column(
```

Comment:
I'd maybe suggest renaming this to `get_column` and get rid of the currently existing `get_column`. It's providing an indirection that's not adding anything since the callable has a default value.

Reply:
I think at this point you may be right. In the future, we might need to add back this indirection if we have more complicated data access patterns, but for now, this additional step is unnecessary perhaps.

Reply:
Sure, once we have the need this could be changed.